application/libraries/template.php

```diff
@@ -468,17 +468,20 @@ public function getSlider()
 		// Load the slides from the database
 		$slides_arr = $this->CI->cms_model->getSlides();
 
-		foreach($slides_arr as $key=>$image)
+		if (is_array($slides_arr) || is_object($slides_arr))
```

Comment:
Maybe add an isset($slides_arr) as well for both of these to ensure they actually exist. Other than that LGTM

Reply:
Any update on this? @illunix

Reply:
> Maybe add an `isset($slides_arr)` as well for both of these to ensure they actually exist. Other than that LGTM

You mean something like that?

Reply:
@mdjong1